pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "django-ibmi"
+version = "0.1.0-dev"
+description = "'Db2 for i support for Django framework'"
+authors = [
+    "Naveen Ram  <naveen.ram@ibm.com>",
+    "Kevin Adler <kadler@us.ibm.com>"
+]
+maintainer='IBM i Open Source Software Team'
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["django", "web framework", "ibm", "ibmi", "db2"]
+packages = [
+    {include = "ibm_db_django"}
+]
+
+[tool.poetry.dependencies]
+python = ">=2.7"
+Django = ">=2.0.0"
```

Comment:
Let's require 2.2+, since that's the most recent 2.x release and also an LTS version.